File: Programmers/Level1/서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        String answer="";
        String a="김서방은 ";
        String b="에 있다";
        for(int i=0; i<seoul.length; i++){
            if(seoul[i].equals("Kim")){
               answer= a + Integer.toString(i) + b;
                break;
            }
        }
        return answer;
    }
}

File: Programmers/Level1/서울에서 김서방 찾기/서울에서 김서방 찾기.go
import "fmt"
func solution(seoul []string) string {
    length:= len(seoul)
    var i int
    for i=0; i<length; i++{
        if seoul[i] == "Kim"{
            break
        }
    }
    
    return fmt.Sprintf("김서방은 %d에 있다", i)
}
